topics/perovskite/index.md

```diff
@@ -0,0 +1,9 @@
+---
+aliases: perovskite
```

Comment:
You shouldn't need to specify `aliases:` since this is the topic itself.

Reply:
ok, I get the point.